### ISSUE TITLE
Family Page SVG heading does not have accessible title

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#12832]
## Description

Meaningful visual content needs text equivalent. Caught by [WAVE](https://wave.webaim.org/extension/) accessibility check

### Steps to reproduce

Run WAVE extension on https://www.mozilla.org/en-US/firefox/family/

### Expected result

No errors.

### Actual result

Error for missing accessible title (also viewable in Firefox Accessibility dev tools)
![Screenshot 2023-03-02 at 9 55 21 AM](https://user-images.githubusercontent.com/19650432/222397833-0d91a170-6490-4fdd-8bfa-5b5af837ef7e.png)

### Environment

N/A
